main.go

```diff
@@ -216,10 +214,19 @@ func init() {
 		GroupID: groupDev,
 		RunE:    withValidProjectURI(cmd.SessionCreateCmd),
 	}
+
 	newCmd.Flags().StringVarP(&config.BuildID, "image", "i", "", "Build ID of the container image to use")
 	newCmd.Flags().StringVar(&config.Provider, "provider", "", "Provider to use")
-	newCmd.Flags().StringVar(&config.NodeTypeID, "type", "", "Node type to use, eg. `gpu_1x_a100`")
+
+	newCmd.Flags().IntVar(&config.GPUs, "gpus", 0, "Number of GPUs to allocate for a gpuType, e.g., 2")
+	newCmd.Flags().IntVar(&config.GPUMemory, "gpuMem", 0, "Memory of GPU if applicable for a gpuType, e.g., 12")
+	newCmd.Flags().StringVar(&config.GPUType, "gpuType", "", "Type of GPU to use, e.g., rtx_5000")
```

Comment:
We use the dash convention for flags. `gpuMem -> gpu-mem`, `gpuType -> gpu-type`